notebooks/11_temporal_probability_models/index.md

```diff
@@ -0,0 +1,603 @@
+
+Figure 2 shows the result of EKF localization for the simple problem given in Figure 1. The ground truth of the robot poses and the estimated robot poses are shown in red and blue, respectively. The 95% confidence ellipses obtained from the covariance matrices in the EKF estimation process are also shown in the figure.
+| ![Figure 1](https://onlinelibrary.wiley.com/cms/asset/7c427b65-cce0-4158-be66-717b00a419ac/nfg002.gif) | 
+|:--:| 
+| *Figure 1* |
+
+![Figure 2](https://onlinelibrary.wiley.com/cms/asset/beb02148-6fb8-4080-b76e-6770ad04f8ff/nfg004.gif) | 
+|:--:| 
+| *Figure 2* |
+
+# Dynamic Bayes Nets
+A Bayesian network is a snapshot of the system at a given time and is used to model systems that are in some kind of equilibrium state. Unfortunately, most systems in the world change over time and sometimes we are interested in how these systems evolve over time more than we are interested in their equilibrium states. Whenever the focus of our reasoning is change of a system over time, we need a tool that is capable of modeling dynamic systems.
+
+A dynamic Bayesian network (DBN) is a Bayesian network extended with additional mechanisms that are capable of modeling influences over time. The temporal extension of Bayesian networks does not mean that the network structure or parameters changes dynamically, but that a dynamic system is modeled. In other words, the underlying process, modeled by a DBN, is stationary. A DBN is a model of a stochastic process.
+
+## DBN particle filtering
+**Basic idea:** ensure that the population of samples (“particles”) tracks the high-likelihood regions of the state-space Replicate particles proportional to likelihood for $e_t$
+
+![Figure 1](https://s4.uupload.ir/files/fig2_m8pf.jpg) | 
+|:--:| 
+| *DBN Particle Filtering* |
+
+Widely used for tracking nonlinear systems, esp. in **vision**. Also used for simultaneous localization and mapping in mobile robots $10^{-5}$ dimensional state space. 
+Assume consistent at time $t: \frac{N(x_t|e_{1:t})}{N}=P(x_t|e_{1:t})$.
+**Propagate forward**: populations of $x_{t+1}$ are
+
+$$
+N(x_t|e_{1:t})=\sum_{x_t} P(x_{t+1}|x_t)N(x_t|e_{1:t})
+$$
+
+**Weight** samples by their likelihood for $e_{t+1}$:
+
+$$
+W(x_{t+1}|e_{1:t+1})= P(e_{t+1}|x_t)N(x_t|e_{1:t})
+$$
+
+**Resample** to obtain populations proportional to $W$:
+
+$$
+\begin{align*}
+\frac{N(x_{t+1}|e_{1:t+1})}{N} &=\alpha W(x_{t+1}|e_{1:t+1}) = \alpha P(e_{t+1}|x_{t+1})N(x_{t+1}|e_{1:t}) \\
+&=\alpha P(e_{t+1}|x_{t+1})\sum_{x_t} P(x_{t+1}|x_t)N(x_t|e_{1:t}) \\
+& = \alpha' P(e_{t+1}|x_{t+1})\sum_{x_t} P(x_{t+1}|x_t)P(x_t|e_{1:t}) \\
+& = P(x_{t+1}|e_{1:t+1})
+\end{align*}
+$$
+
+Approximation error of particle filtering remains bounded over time, at least empirically—theoretical analysis is difficult.
+| ![error of particle filtering](https://s4.uupload.ir/files/fig1_llkv.jpg) | 
+|:--:| 
+| *Error of DBN particle filtering.* |
+
+# Conclusion
```

Comment:
have a more detailed conclusion, the body of your notebook is very long, and I think some important points must be missing here